### PR TITLE
docs - now

### PR DESCRIPTION
**Pull request by nllho**
- Add `now` doc.
- Update examples for related datetime functions.

To do:
- [x] Formatting
- [x] Check website links

### CODE REVIEW

File: docs/questions/query-builder/expressions/now.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Now
+---
+
```
Comment (nllho) [inline docs/questions/query-builder/expressions/now.md:4]: @calherries  

Could you give this doc a skim for correctness?

The "Limitations" section in particular.

We put it through one review when we did the datetime docs for 45, so just want to make sure this info is up to date.

Thanks :)

Comment (jeff-bruemmer) [reply docs/questions/query-builder/expressions/now.md:4]: Do we have any other functions that don't require an invocation (e.g., `now` works without the parens to call it `now()`.)

File: docs/questions/query-builder/expressions/datetimesubtract.md [hunk 2]
```diff
@@ -47,24 +47,22 @@ Here, **Depart At** is a custom column with the expression:
 datetimeSubtract([Arrive By], 30, "minute")
 ```
 
-## Comparing a date to a window of time
+## Checking if the current datetime is within an interval
```
Comment (jeff-bruemmer) [inline docs/questions/query-builder/expressions/datetimesubtract.md:50]: Nice job integrating `now` into existing docs.

File: docs/questions/query-builder/expressions/now.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+| [Data type](https://www.metabase.com/learn/databases/data-types-overview#examples-of-data-types) | Works with `now`  |
+| ----------------------- | -------------------- |
+| String                  | ❌                   |
+| Number                  | ❌                   |
+| Timestamp               | ✅                   |
+| Boolean                 | ❌                   |
+| JSON                    | ❌                   |
+
+This table uses `timestamp` and `datetime` interchangeably. If your dates and times are stored as strings or numbers in your database, you can [cast them to datetimes](../../../data-modeling/metadata-editing.md#casting-to-a-specific-data-type) from the Data Model page.
```
Comment (jeff-bruemmer) [inline docs/questions/query-builder/expressions/now.md:55]: Do we need to get into datetimes with timezones vs without?

File: docs/questions/query-builder/expressions/now.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+`now` might not actually be _now_ (in your local time) if you don't live in the same timezone as your Metabase [report time zone](../../../configuring-metabase/localization.md#report-timezone).
+
+If you need to compare `now` to a column in a different time zone, use [convertTimezone](./converttimezone.md) to shift both columns into the same time zone. For example:
+
+```
+convertTimezone(now, 'UTC', <report timezone>) >= convertTimezone([Deadline], 'UTC', <source time zone>)
+```
+
+`now` is only available on Mongo versions 4.2 or greater.
```
Comment (jeff-bruemmer) [inline docs/questions/query-builder/expressions/now.md:67]:
```suggestion
### Note on `now` in MongoDB

`now` is only available on Mongo versions 4.2 or greater.
```

File: docs/questions/query-builder/expressions/now.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+## Related functions
+
+Different ways to do the same thing, because while you'd love to use custom expressions more, now's just not the time.
```
Comment (jeff-bruemmer) [inline docs/questions/query-builder/expressions/now.md:71]: > now's just not the time.

Chef's kiss.

**Review by jeff-bruemmer — APPROVED** [review]
Nice work, @nllho.

File: docs/questions/query-builder/expressions/now.md (hunk 1 above)
Comment (calherries) [reply docs/questions/query-builder/expressions/now.md:4]: @jeff-bruemmer we have `count` and `sum` for summarize custom expressions.

File: docs/questions/query-builder/expressions/now.md (hunk 4 above)
Comment (calherries) [reply docs/questions/query-builder/expressions/now.md:67]: We don't support Mongo versions <4.2 anymore, so we can remove this note. https://github.com/metabase/metabase/blob/3e636378c539cbeec5b4d3a47e0d7f9605207d7f/docs/databases/connections/mongodb.md#L9

File: docs/questions/query-builder/expressions/now.md (hunk 3 above)
Comment (calherries) [reply docs/questions/query-builder/expressions/now.md:55]: Yes, I think it matters. But I think it's sufficient to state that for DBMSs that support timestamps with time zones, `now` is a timestamp with a time zone that is the reporting time zone. For others, it's a timestamp (without time zone) in the  reporting time zone.

Comment (calherries) [reply docs/questions/query-builder/expressions/now.md:55]: It's mostly implied in the decscription, but it's not crystal clear
https://github.com/metabase/metabase/pull/29182/files#diff-2d5b6a8a31092835d5e0c3a68db37209bd715411ab0644950fb7268bb3b88fe8R436

**Pull request merged**